website/docs/reference/dbt_project.yml.md

```diff
@@ -1,8 +1,9 @@
 ---
 description: "Reference guide for configuring the dbt_project.yml file."
+intro_text: "The dbt_project.yml file is a required file for all dbt projects. It contains important information that tells dbt how to operate your project."
```

Comment:
What's our best practice for intro text? Should it be describing the feature or describing what's on the page (for example "This is the reference page for configuring the dbt_project.yml, a required file for all dbt projects that contains important information that tells dbt how to operate your project."

Reply:
It's a little bit of both I'd say, intro text is probably a little longer than description